### ISSUE TITLE
Ideia: Drag in compose

### ISSUE THREAD

**Issue by kalicki** [lavab/web#864]
![untitled screencast](https://cloud.githubusercontent.com/assets/3674885/8170470/28f85e4a-1387-11e5-85be-d748c81003f8.gif)

So give me opportunities to view other emails to streamline in the writing of the new

**Comment by kalicki** [comment]
![lavaboomideia1](https://cloud.githubusercontent.com/assets/3674885/8207763/07d43b36-14da-11e5-9dba-6c4e0d6d3bc4.png)

**Comment by kalicki** [comment]
Right click actions in email ( e.g. http://ianwalter.github.io/ng-context-menu/#/)

**Comment by piggyslasher** [comment]
I don't think we're going to support right click. It just not usual for a
webapp to have right clicks these days. Will keep this issue open till its
discussed.

**Comment by kalicki** [comment]
@piggyslasher Web-only and on large screens (dekstop, laptop, ...) is very useful, to mobile and tablet doesn't make sense

**Comment by piggyslasher** [comment]
I was thinking the same thing. But you have to understand the fact that no
modern webapps use a right click. What we can do is add it to a menu that
you click on the open.

**Comment by piggyslasher** [comment]
It's going to get much easier. But a right click probably won't happen

**Comment by piggyslasher** [comment]
And Mobile right click doesn't happen. Unless you're talking about the
press & hold.

**Comment by kalicki** [comment]
Yes, that's why I quoted doesn't make sense for mobile, but ok, that's not important! Multiple actions are most important hahaha

**Comment by kalicki** [comment]
Multiple actions in order to select multiple emails to delete, archive, ...

**Comment by kalicki** [comment]
@piggyslasher But the rest of the ideas you find interesting?
